semperfiwebdesign/all-in-one-seo-pack
# Make webmaster tools strings untranslatable

Issue #2669

## Proposed changes

Makes webmaster tools strings untranslatable with the exception of the Pinterest one.

## Types of changes

- Improves existing code

## Checklist


## Testing instructions

Install the PR and go to the General Settings menu to make sure these strings still look good.
